healthcareai/supervised_model_trainer.py

```diff
@@ -78,9 +78,15 @@ def knn(self):
         print('\nTraining {}'.format(model_name))
         t0 = time.time()
```

Comment:
Why don't we prototype moving all the shared timing code into the advanced trainer class so it can be consolidated into a single place. That would even further simplify the trainer class